src/themes/dojo/toolbar.m.css

```diff
@@ -0,0 +1,38 @@
+@import './variables.css';
+
+.toolbar {
+	background: #FFF;
+	border: var(--border-width) solid var(--color-border);
+	box-sizing: border-box;
+	height: calc(7 * var(--grid-base));
+	padding: calc(2 * var(--grid-base));
+	text-align: left;
+}
+
+.title {
+	font-size: var(--font-size-title);
+}
+
+.action {
+	padding-right: calc(3 * var(--grid-base));
+}
+
+.action:last-of-type {
+	padding-right: 0;
+}
+
+.menuButton {
+	background: none;
+	border: none;
+	cursor: pointer;
+	font-size: 0;
+	padding: var(--grid-base);
+	position: absolute;
+	right: var(--grid-base);
+	top: 50%;
```

Comment:
Sort of, but it would require one extra element. Since I'm already absolutely positioning the button to the right it was a bit cleaner (and more uniform) to just use `top` and `transform`.

Reply:
what about when our users want to use flexbox? If adding an extra element to make this simpler and removes absolute positioning etc is it not worth it?